examples/spring/pom.xml

```diff
@@ -8,7 +8,7 @@
         <relativePath>..</relativePath>
     </parent>
 
-    <artifactId>spring-example</artifactId>
+    <artifactId>spring</artifactId>
```

Comment:
lets keep the previous name as it is more meaningful

Reply:
@dkuc84 Do you want to change the name of the folders all together? This is just the maven module name and i think it makes sense that it should match the directory name

If we want to change that I would do that in a separate PR

Reply:
artifact ID often match the folder name but by no means it is a requirement. `spring-example` is much more meaningful artifact name than just the `spring`. Up to you whether you want to rename the folder to match the artifact ID but lets keep the module name as `spring-example`.

Reply:
I would prefer changing the name of the directory, I will do that as another PR

Reply:
I will undo my change here any if we need to make updates later we can